pipeline/variables.sample.yml

```diff
@@ -1,61 +1,114 @@
+  
 #
 # Terraform
 #
 
 # Amazon access to use to run terraform. Usually admin account to be able to create Amazon services.
+
+#. aws_access_key (required): ((aws.access_key))
+#+ Amazon AWS access key for Terraform. see value format [Here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline)
 aws_access_key: ((aws.access_key))
+
+#. aws_secret_key (required): ((aws.secret_key))
+#+ Amazon AWS secret key for Terraform. see value format [Here](https://docs.cycloid.io/advanced-guide/integrate-and-use-cycloid-credentials-manager.html#vault-in-the-pipeline)
 aws_secret_key: ((aws.secret_key))
-aws_default_region: eu-west-1
 
-# Password used for your rds. Set "empty" if you dont use databases
-rds_password: ((raw_rds_password))
+#. aws_default_region (required): eu-west-1
+#+ Amazon AWS region to use for Terraform.
+aws_default_region: eu-west-1
 
-# Bucket configuration to store terraform remote state file.
+#. terraform_storage_bucket_name (required): ($ organization_canonical $)-terraform-remote-state
+#+ AWS S3 bucket name to store terraform remote state file.
```

Comment:
Shall we re-comment that those variables `($ organization_canonical $)` are replaced by the product, or shall we simply point to our documentation link?

Reply:
If it's not clear, it will be up to us to make it more clear in the doc or frontend display